docs/excel/excel-add-ins-pivottables.md

```diff
@@ -221,7 +221,128 @@ Excel.run(function (context) {
+
+### Filter with PivotFilters
+
+[PivotFilters](/javascript/api/excel/excel.pivotfilters) allow you to filter PivotTable data based on four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). In the PivotTable object model, `PivotFilters` are contained within [PivotHierarchies](/javascript/api/excel/excel.pivothierarchy). A PivotFilter can only be applied to a PivotHierarchy that is used to pivot data within the table.
+
+#### Types of PivotFilters
+
+| Filter type | Filter purpose | Excel JavaScript API reference |
+|:--- |:--- |:--- |
+| DateFilter | Calendar date-based filtering. | [PivotDateFilter](/javascript/api/excel/excel.pivotdatefilter) |
+| LabelFilter | Text comparison filtering. | [PivotLabelFilter](/javascript/api/excel/excel.pivotlabelfilter) |
+| ManualFilter | Custom input filtering. | [PivotManualFilter](/javascript/api/excel/excel.pivotmanualfilter) |
+| ValueFilter | Number comparison filtering. | [PivotValueFilter](/javascript/api/excel/excel.pivotvaluefilter) |
+
+#### Create a PivotFilter
+
+The following four code samples show how to use each filter described in the preceding [Types of PivotFilters](#types-of-pivotfilters) table. The first code sample applies a [PivotDateFilter](/javascript/api/excel/excel.pivotdatefilter) to the **Date Updated** hierarchy, hiding any data prior to a specified date. 
+
+> [!IMPORTANT] 
+> The `dateHierarchy` must be added to the PivotTable's `rowHierarchies` before it can be used for filtering.
+
+```js
+Excel.run(function (context) {
+    // Get the PivotTable and the date hierarchy.
+    var pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+    var dateHierarchy = pivotTable.rowHierarchies.getItemOrNullObject("Date Updated");
+    
+    return context.sync().then(function () {
+        // PivotFilters can only be applied to PivotHierarchies that are being used for pivoting.
+        // If it's not already there, add "Date Updated" to the hierarchies.
+        if (dateHierarchy.isNullObject) {
+          dateHierarchy = pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Date Updated"));
+        }
+
+        // Apply a date filter to filter out anything logged before August.
+        var filterField = dateHierarchy.fields.getItem("Date Updated");
+        var dateFilter = {
+          condition: Excel.DateFilterCondition.afterOrEqualTo,
+          comparator: {
+            date: "2020-08-01",
+            specificity: Excel.FilterDatetimeSpecificity.month
+          }
+        };
+        filterField.applyFilter({ dateFilter: dateFilter });
+        
+        return context.sync();
+    });
+});
+```
+
+> [!NOTE]
+> The three subsequent code snippets display only filter-specific excerpts and are not complete code samples. See the preceding code sample for a complete `PivotFilter` implementation example.
```

Comment:
It's hard to define what a "complete" code sample would be (since all these snippets aren't a complete application). Consider 
```suggestion
> The three subsequent code snippets only display filter-specific excerpts, instead of full `Excel.run` calls.
```

@Rick-Kirkham, do you have any thoughts about how to frame these smaller code snippets (if at all)?